build/VStudio/dotnet/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
Is this file necessary? You can use file `build.common.props` to set properties. But in this case I would prefer to keep these properties in the `.csproj` file to minimize changes in this PR.

Reply:
Outside reader, This file automatically applies to all projects in all subdirectories. the csproj files will in the end be close to empty.

Reply:
@NiKiZe I understand the purpose of the `Directory.Build.props` file. I just think it is unnecessary because the WinFsp project already has its own `build.common.props` file for the same purpose and because I would prefer to minimize changes by keeping the properties in the `.csproj` files for now.